ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/platform.j2

```diff
@@ -4,6 +4,43 @@
 {%     if platform.trident.forwarding_table_partition is arista.avd.defined %}
 platform trident forwarding-table partition {{ platform.trident.forwarding_table_partition }}
 {%     endif %}
+{%     if platform.trident.mmu.active_profile is arista.avd.defined %}
+platform trident mmu queue profile {{ platform.trident.mmu.active_profile }} apply
+{%     endif %}
+{%     for profile in platform.trident.mmu.queue_profiles | arista.avd.natural_sort('name') %}
+!
+platform trident mmu queue profile {{ profile.name }}
+{%         for queue in profile.multicast_queues | arista.avd.natural_sort('id') %}
+{%             if queue.reserved is arista.avd.defined %}
+{%                 set unit = "" %}
+{%                 if queue.unit is arista.avd.defined %}
+{%                     set unit = " " ~ queue.unit | arista.avd.default('') %}
+{%                 endif %}
+    egress multicast queue {{ queue.id }} reserved{{ unit }} {{ queue.reserved }}
```

Comment:
I would prefer to use the same style as we do elsewhere:
```suggestion
{%             if queue.reserved is arista.avd.defined %}
{%                 set reserved_cli = "egress multicast queue " ~ queue.id ~ " reserved" %}
{%                 if queue.unit is arista.avd.defined %}
{%                     set reserved_cli = reserved_cli ~ " " ~ queue.unit %}
{%                 endif %}
{%                 set reserved_cli = reserved_cli ~ " " ~ queue.reserved %}
    {{ reserved_cli }}
```

Reply:
Agreed, updated the template.